templates/p5_gfd.bt

```diff
@@ -119,8 +125,8 @@ typedef struct(u32 version)
     {
         char Str[Length];  
     
-        if ( version >= 0x01105100 )
-            FSeek( FTell() + 1 ); // padding byte
+        /*if ( version >= 0x01105100 )
```

Comment:
why is this commented out

Reply:
This needs some sort of better implementation.

This check exists for Catherine Full Body support, however Persona 5 Royal shares the same resource version while not having this extra byte, meaning this either makes CFB work while not making P5R work, or make P5R work and not CFB, current workaround would be have users renumber the resource version to a lower version.